### PR TITLE
Credit Note, Debit Note, Excise Entry info added

### LINKED ISSUE #6512: Journal Entry
- [x] Add details about debit note, credit note, excise entry details with example 
- [ ] Add video of any use case

### PR DESCRIPTION
**Pull request by KanchanChauhan**
Fixed https://github.com/frappe/erpnext/issues/6512

### CODE REVIEW

File: erpnext/docs/user/manual/en/accounts/journal-entry.md [hunk 1]
```diff
@@ -68,4 +68,38 @@ of each year reducing its value by a certain percentage.
...
+#### Debit Note
+
+Debit note is a note which is given to the supplier or accepted as a credit note 
+from our supplier when we return goods. By giving this note, we try to tell that 
+we are debiting his account with returned goods amount. 
+
+  * Debit: Supplier Account
+  * Credit: Purchase Return Account 
+
+#### Excise Entry
```
Comment (rmehta) [inline erpnext/docs/user/manual/en/accounts/journal-entry.md]: This is only for India, maybe this should be a separate article (maybe we should have separate sections for country-wise pages)

File: erpnext/docs/user/manual/en/accounts/journal-entry.md [hunk 2]
```diff
@@ -68,4 +68,38 @@ of each year reducing its value by a certain percentage.
 > Note: There may be regulations in your country that define by how much
 amount you can depreciate a class of Assets.
 
+#### Credit Note
+
+Credit note is a note which is given to the customer when we get his returned
```
Comment (rmehta) [inline erpnext/docs/user/manual/en/accounts/journal-entry.md]: Try and use gender neutral language, also documentation is best written in 3rd person.

`"Credit Note" is made for a Customer against a Sales Invoice when the company needs to refund or adjust a payment. When a Credit Note is made, the seller can either make a payment to the customer or adjust the amount in another invoice.`

File: erpnext/docs/user/manual/en/accounts/journal-entry.md [hunk 3]
```diff
@@ -68,4 +68,38 @@ of each year reducing its value by a certain percentage.
...
 
+#### Credit Note
+
+Credit note is a note which is given to the customer when we get his returned 
+goods. By giving this note to customer, we try to tell that we are crediting his 
+account with his returned goods amount.
+
+  * Debit: Sales Return Account
+  * Credit: Customer Account
+
```
Comment (rmehta) [inline erpnext/docs/user/manual/en/accounts/journal-entry.md]: Screenshot would be better!

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/docs/user/manual/en/accounts/journal-entry.md (hunk 2 above)
Comment (KanchanChauhan) [inline erpnext/docs/user/manual/en/accounts/journal-entry.md]: Sales Invoice or Sales Return?

File: erpnext/docs/user/manual/en/accounts/journal-entry.md (hunk 1 above)
Comment (KanchanChauhan) [inline erpnext/docs/user/manual/en/accounts/journal-entry.md]: Added a note, since option is available for other countries as well.

**Review by nabinhait — APPROVED** [review]

**Pull request merged**